docs/rackhd/event_notification.rst

```diff
@@ -1,105 +1,264 @@
 Event Notification
 ------------------
 
-The change of resources managed by RackHD could be retrieved from AMQP messages.
-
-- Exchange: **on.events**
-- Routing Key **event.<type>**
+RackHD support event notification via both AMQP and web hook.
 
-`<type>` is event's type, it's the same with event's payload `type`, see payloads_ for more details.
+Web hook allow user's application to subscribe certain RackHD published events published by configured URL, when one of the subscribed events is triggered, RackHD will send a HTTP POST request with payload to configured URL.
```

Comment:
an example is in the following section 'curl -H "Content-Type: application/json" -X POST -d @payload.json <server>/api/2.0/hooks' 
or do you mean an example of POST an event payload to hook URL?